src/drivers/pci.rs

```diff
@@ -29,18 +34,26 @@ use crate::console::IoDevice;
 use crate::drivers::console::{VirtioConsoleDriver, VirtioUART};
 #[cfg(feature = "fuse")]
 use crate::drivers::fs::virtio_fs::VirtioFsDriver;
-#[cfg(any(feature = "tcp", feature = "udp"))]
+#[cfg(all(
+	any(feature = "tcp", feature = "udp"),
+	any(
+		all(target_arch = "x86_64", feature = "rtl8139"),
```

Comment:
Why do we need both `feature = "rtl8139"` and `target_arch = "x86_64"`? Would the former not be enough?